procedures/candlepin/branch.md.erb

```diff
@@ -0,0 +1,21 @@
+# Branch Packaging
+
+- [ ] Clone https://github.com/theforeman/candlepin-packaging
+- [ ] Check out most recent branch `rpm/< latest_version >`
+- [ ] Create a new branch for the version being added
+  - `git checkout -b rpm/<%= release %>`
```

Comment:
You can skip the recent branch checkout:
```suggestion
  - `git checkout -b rpm/<%= release %> rpm/develop`
```

Though I typically prefer to use a remote to ensure it's latest without having to pull. A more complete script that clones or updates:
```
REMOTE=origin
git clone https://github.com/theforeman/candlepin-packaging -o "$REMOTE" || ( cd candlepin-packaging && git fetch "$REMOTE" )
cd candlepin-packaging
git checkout -b rpm/<%= release %> "$REMOTE/rpm/develop"
```

A more generalized version is available in the repo:
https://github.com/theforeman/theforeman-rel-eng/blob/master/branch_project